app/views/providers/allocations/number_of_places.html.erb

```diff
@@ -7,13 +7,14 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
 
-  <%= form_with url: initial_request_provider_recruitment_cycle_allocations_path(
+  <%= form_with  model: form_object, scope: "", url: initial_request_provider_recruitment_cycle_allocations_path(
                     @provider.provider_code,
                     @provider.recruitment_cycle_year,
                   ),
                 skip_enforcing_utf8: true,
-                method: :get,
+                # method: :get,
```

Comment:
delete it or leave it otherwise the next person will be confused

Reply:
oops ... should have deleted